cmd/influx/global.go

```diff
@@ -271,9 +271,9 @@ OPTIONS:
 		customFunc := make(map[string]interface{})
 		customFunc["iscommon"] = func(flag cli.Flag) bool {
 			switch flag.(type) {
-			case CommonBoolFlag:
+			case *CommonBoolFlag:
```

Comment:
Oops - thanks! Must not have tested the final version after switching to the switch statement :(